dockers/docker-syncd-brcm-rpc/Dockerfile

```diff
@@ -43,7 +43,7 @@ RUN apt-get update \
 COPY supervisord.conf /etc/supervisor/conf.d/supervisord.conf
 COPY ptf_nn_agent.conf /etc/supervisor/conf.d/ptf_nn_agent.conf
 
-ENTRYPOINT service rsyslog start \
-    && sysctl -w net.core.rmem_max=509430500 \
-    && service syncd start       \
-    && /usr/bin/supervisord
+RUN sed -i 's/read/sysctl -w net.core.rmem_max=509430500 ; \/usr\/bin\/supervisord/' /usr/bin/start.sh
```

Comment:
I like what you're doing here, but all of the other start.sh files are checked in to the project. Just wondering if you should follow suit here for uniformity's sake. If the file is present in the repo, it's obvious it should be used.